schemas/oracleSource.js

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Could you not commit the schema's that we are going to use.  Let's keep this just as a tool and use those schema's outside of this repo.